cmd/glasskube/cmd/bootstrap.go

```diff
@@ -31,6 +32,30 @@ var bootstrapCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		cfg, _ := cliutils.RequireConfig(config.Kubeconfig)
 		client := bootstrap.NewBootstrapClient(cfg)
+
+		installedVersion, err := clientutils.GetPackageOperatorVersion(cmd.Context())
+		if err != nil {
+			IsBootstrapped, err := bootstrap.IsBootstrapped(cmd.Context(), cfg)
+			if err != nil && !IsBootstrapped {
+				fmt.Printf("error : %v\n", err)
+				cliutils.ExitWithError()
+			}
+		}
+
+		var desiredVersion string
+		if bootstrapCmdOptions.url == "" {
+			desiredVersion = config.Version
+		} else {
+			desiredVersion = ""
+		}
+		if !cliutils.YesNoPrompt(fmt.Sprintf("Glasskube is already installed in this cluster "+
```

Comment:
@hanshal101 where did the `semver.IsUpgradable` part go? the prompt should only be visible if `installedVersion != "" && !semver.IsUpgradable(...)`